docker-compose/integrator-broker-analytics/docker-compose.yml

```diff
@@ -86,7 +86,7 @@ services:
       - wso2ei-mysql
       - wso2ei-analytics
   wso2ei-integrator:
-    image: docker.wso2.com/wso2ei-integrator:6.1.1
+    image: wso2ei-integrator:6.2.0
```

Comment:
Why is the docker.wso2.com prefix missing?

Reply:
@msmshariq will fix it. Couldn't revert the change after testing was finished.